### ISSUE TITLE
Is esp32 faster the esp8266

### ISSUE THREAD

**Issue by wdouglas1999** [ArduCAM/ArduCAM_ESP32S_UNO#4]
Good day. What is the maximum frame rate that can be achieved with the esp32 compared to the 8266 board?
Is there any improvement in performance other than more I/O pins?

**Comment by ArduCAM** [comment]
With our experience, ESP32 is much more powerful than ESP8266.
The frame rate depends on resolution and image size. It can reach up to 7fps@QVGA MJPEG on ESP32.

**Comment by wdouglas1999** [comment]
Hi, this is about the same as esp8266. So the is no advantage to have this board other than more gpios.

**Comment by ArduCAM** [comment]
I don't think so, ESP32 is much faster than ESP8266, ESP32 has 200MHz+ dual core MCU and supports both WIFI and BLE as well.
